src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx

```diff
@@ -179,7 +179,13 @@ const SupportLevel = ({ cluster }: SupportLevelProps) => {
 };
 
 const ClusterFeatureSupportLevelsDetailItem = ({ ...props }: SupportLevelProps) => (
-  <WithErrorBoundary title="Failed to load feature support levels review">
+  <WithErrorBoundary
+    title="Feature support levels not available"
+    content={`OpenShift version ${
+      props.cluster.openshiftVersion || ''
+    } does not currently review feature support levels.`}
```

Comment:
we need microcopy help here

Reply:
What microcopy are we missing?

Reply:
We could generate this message: "OpenShift version does not currently review feature support levels".
Sounds a bit weird...

Reply:
Type-wise yes, we could. But in reality I don't think we'll ever have a cluster without an Openshift version on the installation page. So the  `|| ''` part is really there only to make the linters happy.

Reply:
ack